css/theme.css

```diff
@@ -129,4 +128,7 @@ body {
     height: 40vw;
     width: 63vw
     }
+    .dropbtn{
+        z-index: -1;
```

Comment:
I think `z-index` should be positive. Right now, album tiles are overlapping the songs. We need the opposite.